councilmatic_core/management/commands/refresh_pic.py

```diff
@@ -10,18 +10,21 @@
 from django.db.models import Q
 import pytz
 
-from opencivicdata.legislative.models import BillDocumentLink, EventDocumentLink, \
-    EventRelatedEntity
+from opencivicdata.legislative.models import BillDocumentLink, BillVersionLink, \
+    EventDocumentLink, EventRelatedEntity
 
 
 for configuration in ['AWS_KEY','AWS_SECRET']:
     if not hasattr(settings, configuration):
         raise ImproperlyConfigured(
             'Please define {0} in settings_deployment.py'.format(configuration))
 
+
 logging.config.dictConfig(settings.LOGGING)
 logger = logging.getLogger(__name__)
 
+app_timezone = pytz.timezone(settings.TIME_ZONE)
```

Comment:
This feels like maybe it would be appropriate as an attribute in an `__init__()` method as well?

Reply:
Great point! Moved.